curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 2
+Height will automatically adapt to fit all the floated elements.
 
 ---
 
-Placeholder distractor 3
+The containing block will automatically apply `clear: both` to fit the floated element 
 
 #### --answer--
 
-Placeholder answer
+The containing block will collapse, completely ignoring its floated element height.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following CSS techniques is an accepted method for preventing a parent container from schrinking in size when it contains only floated elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+Set `float: none` to the container
 
 ---
 
-Placeholder distractor 2
+Add `margin: auto` to the parent container.
 
 ---
 
-Placeholder distractor 3
+Use the `position: relative` on the parent container 
 
 #### --answer--
 
-Placeholder answer
+Adding the property `overflow: auto` or `overflow: hidden` to the parent container
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When a floating element is followed by non-floating inline elements, it will affect the way in which their layout is performed. 
 
 #### --distractors--
 
-Placeholder distractor 1
+The inline elements will be positioned below the floated element. 
 
 ---
 
-Placeholder distractor 2
+Inline elements will always be on top of the floated element
 
 ---
 
-Placeholder distractor 3
+The inline elements will overlap the floated element 
 
 #### --answer--
 
-Placeholder answer
+Inline elements will wrap around the floated element if there is enough space.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when clear: both is applied to an element following two floated elements -one left and one right - in a container?
 
 #### --distractors--
 
-Placeholder distractor 1
+The element will be positioned to the left of the floated elements.
 
 ---
 
-Placeholder distractor 2
+The element will align between the two floated elements
 
 ---
 
-Placeholder distractor 3
+The element will wrap around the floated elements depending on space 
 
 #### --answer--
 
-Placeholder answer
+The element will be underneath both floated elements - the float breaks 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When does an element with z-index: auto create a new stacking context?
 
 #### --distractors--
 
-Placeholder distractor 1
+Combined with `position: absolute` or `position: relative`
 
 ---
 
-Placeholder distractor 2
+When its parent element has `z-index: auto` and `position: fixed`
 
 ---
 
-Placeholder distractor 3
+When it is combined with `float: left` If two sibling elements are of the same `z-index` and absolutely positioned
 
 #### --answer--
 
-Placeholder answer
+In combination with `position: fixed` or `position: sticky`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If two sibling elements have the same z-index and are both positioned absolutely using position: absolute, how do you determine which one is on top?
```

Comment:
```suggestion
If two sibling elements have the same `z-index` and are both positioned absolutely using `position: absolute`, what determins which one is on top?
```

But this isn't what the answer is talking about. It talks about _changing_ the source order. So either this need to change to how you control which one is on top or the answer needs to be changed.